src/main/webapp/Portal.html

```diff
@@ -25,7 +25,7 @@
     />
     <meta
       property="og:image"
```

Comment:
```suggestion
      name="image"
      property="og:image"
```

Reply:
You're too fast at reviewing :)
Change made.